### DISCUSSION TITLE
organization has been flagged.

### DISCUSSION THREAD

**Discussion by UNITSLIMITEDSUPPLY** [community/community#101359] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

My organization faced a problem, it was flagged and hidden. The reasons for this are not explained. Technical support has been silent for 5 days. Github put me in an awkward position in front of society because my website was located here. Please tell me who can help me with this?
![Screenshot_5](https://github.com/community/community/assets/151500812/e8fc0dda-c054-4c25-8142-f88224150cb5)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by UNITSLIMITEDSUPPLY** [comment]
Still waiting

**Comment by queenofcorgis** [comment, marked as answer]
You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects.

For your situation, the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
